databricks_cli/configure/config.py

```diff
@@ -51,30 +52,35 @@ def decorator(*args, **kwargs):
     return decorator
 
 
-def _get_api_client():
+def profile_option(f):
```

Comment:
Why don't we just document how to create config file? I don't think the command option is that useful for this. As a user I would be fine to copy paste and modify the token.

Reply:
I think that would be useful as well. Regarding the command option, `aws configure` supports the `--profile` option so this PR follows this convention.